fullscreen.bs

```diff
@@ -396,6 +405,13 @@ could be an open <{dialog}> element.
  <a>simple fullscreen document</a>, then set <var>doc</var> to <var>topLevelDoc</var> and
  <var>resize</var> to true.
 
+ <li><p>If <var>doc</var>'s <a>fullscreen element</a> is not <a>connected</a>:
+  <ol>
+   <li><p><a for=set>Append</a> (<code>fullscreenchange</code>, <var>doc</var>'s
```

Comment:
Hmm, I realized one little wrinkle. We should probably call "unfullscreen an element" on the fullscreen element, so that the fullscreen flags are unset. Otherwise we'll be in a weird state where https://fullscreen.spec.whatwg.org/#css-pc-fullscreen continues to match.

That will mean that the other added "remove node from document’s top layer" will *only* clean up things other than fullscreen. That'll just have to be OK I think.